src/specfit.jl

```diff
@@ -208,31 +212,34 @@ Get the FWHM of a peak from the fit parameters.
 # Returns
     * `fwhm`: the FWHM of the peak
 """
-function estimate_fwhm(v::NamedTuple)
-    
-    f_sigWithTail = Base.Fix2(get_th228_fit_functions().gamma_sigWithTail,v)
-    try
-        e_low, e_high = v.skew_fraction <= 0.5 ? (v.μ - v.σ, v.μ + v.σ) : (v.μ * (1 - v.skew_width), v.μ * (1 + v.skew_width))
-        
-        max_sig = -Inf
-        for e in e_low:0.001:e_high
-            fe = f_sigWithTail(e)
-            if fe > max_sig
-                max_sig = fe
-            else
-                # if the maximum is reached,
-                # no need to further continue
-                break
+function estimate_fwhm(v::NamedTuple) 
+    if haskey(v, :skew_fraction)
+        f_sigWithTail = Base.Fix2(get_th228_fit_functions().gamma_sigWithTail,v)
+        try
+            e_low, e_high = v.skew_fraction <= 0.5 ? (v.μ - v.σ, v.μ + v.σ) : (v.μ * (1 - v.skew_width), v.μ * (1 + v.skew_width))
+            
+            max_sig = -Inf
+            for e in e_low:0.001:e_high
+                fe = f_sigWithTail(e)
+                if fe > max_sig
+                    max_sig = fe
+                else
+                    # if the maximum is reached,
+                    # no need to further continue
+                    break
+                end
             end
+            half_max_sig = max_sig/2
+            
+            tmp = x -> f_sigWithTail(x) - half_max_sig
+            roots_low = find_zero(tmp, e_low, maxiter=100)
+            roots_high = find_zero(tmp, e_high, maxiter=100)
+            return roots_high - roots_low
+        catch
+            return NaN
         end
-        half_max_sig = max_sig/2
-        
-        tmp = x -> f_sigWithTail(x) - half_max_sig
-        roots_low = find_zero(tmp, e_low, maxiter=100)
-        roots_high = find_zero(tmp, e_high, maxiter=100)
-        return roots_high - roots_low
-    catch
-        return NaN
+    else
+        return 2 * sqrt(2 * log(2)) * v.σ
     end
```

Comment:
This right now fails if `v` has no `σ`

Reply:
yes, but there is no gamma fit function that doesn't have a `σ` as far as I know. I'll add a fall-back case anyway to be on the safe site.